ceph.spec.in

```diff
@@ -201,18 +201,24 @@ BuildRequires:  librdkafka-devel
 %if 0%{with make_check}
 BuildRequires:  jq
 BuildRequires:	libuuid-devel
-BuildRequires:	python%{python3_pkgversion}-bcrypt
+BuildRequires:	python%{python3_version_nodots}-bcrypt
```

Comment:
i don't think el8 or el7 offers python36-bcrypt. see https://pkgs.org/download/python36-bcrypt and https://src.fedoraproject.org/rpms/python-bcrypt/branches

Reply:
you are right, el8 does not provide `python36-bcrypt` package. should I ignore this package on el8?

Reply:
i am not sure what you mean by "ignore this package". EPEL8 does offer python3-bcrypt. so i think the existing 

> BuildRequires:	python%{python3_pkgversion}-bcrypt

is correct.

Reply:
The following is the result of my execution in centos7:

```bash
$ yum search python bcrypt
cryptsetup-python.x86_64 : Python bindings for libcryptsetup
python2-bcrypt.x86_64 : Modern password hashing for your software and your servers
python36-bcrypt.x86_64 : Modern password hashing for your software and your servers
```

Should I use if statements to distinguish between el7 and el8?

```ini
%if 0%{?rhel} == 8
BuildRequires:	python%{python3_pkgversion}-bcrypt
%else
BuildRequires:	python%{python3_version_nodots}-bcrypt
%endif
```

Reply:
yeah, i think that's the right way to go.